### PR TITLE
1390712: Add --remove-rhn-packages to man pages

### PR DESCRIPTION
**Pull request by kahowell**
For both rhn-migrate-classic-to-rhsm and sat5to6.

### CODE REVIEW

**Comment by barnabycourt** [comment]
Some of the options conflict with others. Please update the man page with this information. See https://github.com/candlepin/subscription-manager/pull/1517 for details.

**Comment by cnsnyder** [comment]
@kahowell Looks good to me (aside from the requested changes from bcourt). I'll merge after those changes are made.

**Comment by kahowell** [comment]
@cnsnyder, @barnabycourt, fixed. Open to feedback on wording that I added.

File: man/rhn-migrate-classic-to-rhsm.8 [hunk 1]
```diff
@@ -64,6 +64,27 @@ Deletes the system from Red Hat Network Classic and registers it to Customer Por
...
+\fBrhn-setup\fP,
+\fBrhnpush\fP,
+\fBrhnsd\fP,
+\fBspacewalk-abrt\fP,
+\fBspacewalk-oscap\fP,
+\fByum-rhn-plugin\fP.
+Once unregistered from Red Hat Network Classic, they are no longer needed.
+Once packages are removed, subsequent attempts to migrate will be blocked with a message indicating
+"Perhaps this script was already executed with --remove-rhn-packages?".
+\fB--remove-rhn-packages\fP cannot be used with \fB--keep\fP, since it does not make since to keep a system registered without the tooling.
```
Comment (awood) [inline man/rhn-migrate-classic-to-rhsm.8]: "make since" should be "make sense"

File: man/sat5to6.8 [hunk 2]
```diff
@@ -66,6 +66,27 @@ Set a preferred service level for the system, such as premium or standard.
...
+\fBrhn-setup\fP,
+\fBrhnpush\fP,
+\fBrhnsd\fP,
+\fBspacewalk-abrt\fP,
+\fBspacewalk-oscap\fP,
+\fByum-rhn-plugin\fP.
+Once unregistered from Satellite v.5, they are no longer needed.
+Once packages are removed, subsequent attempts to migrate will be blocked with a message indicating
+"Perhaps this script was already executed with --remove-rhn-packages?".
+\fB--remove-rhn-packages\fP can only be used with \fB--registration-state=purge\fP, since it does not make since to keep a system registered without the tooling.
```
Comment (awood) [inline man/sat5to6.8]: "make since" should be "make sense" here too

**Comment by kahowell** [comment]
retest this please

**Comment by cnsnyder** [comment]
LGTM

**Pull request merged**